lib/Data/Chronicle.pm

```diff
@@ -0,0 +1,309 @@
+package Data::Chronicle;
+
+use 5.006;
```

Comment:
It's only tested for 5.14++. Also `Makefile.PL` states min version as 5.14. So ...?